src/app/hackernews-api.service.ts

```diff
@@ -1,28 +1,46 @@
 import { Injectable } from '@angular/core';
-import { Http } from '@angular/http';
+// import { Http } from '@angular/http';
 import { Observable } from 'rxjs/Observable';
 import 'rxjs/add/operator/map';
 
+declare var fetch;
+
+//wrap fetch in observable so we can keep it chill
+export function lazyFetch(url, options?){
```

Comment:
I will fix it.. I replaced http module with this implementation I found on @robwormald repo before testing the case.